pkg/statsd/statsd.go

```diff
@@ -0,0 +1,58 @@
+package statsd
+
+import (
+	"github.com/cactus/go-statsd-client/statsd"
+	"log"
+	"time"
+)
+
+func main() {
+}
+
+type SD struct {
+	Client statsd.Statter
+}
+
+var (
+	client statsd.Statter
+)
+
+func connect(url string) statsd.Statter {
+
+	if client != nil {
+		return client
+	}
+
+	client, err := statsd.NewBufferedClient(url, "pillar", 300*time.Millisecond, 0)
+
+	if err != nil {
+		log.Printf("Error connecting to Statsd [%v]", err)
+		return nil
+	}
+
+	log.Printf("Connected to Statsd")
```

Comment:
We need to start using the same log library everywhere. I have been using the ardanlabs one.